entrypoint/clusterreset/clusterreset_test.go

```diff
@@ -0,0 +1,93 @@
+package clusterreset
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Verifies ClusterIP Service", func() {
+		testcase.TestServiceClusterIp(true, true)
+	})
+
+	It("Verifies NodePort Service Before Reset", func() {
+		testcase.TestServiceNodePort(true, false)
+	})
+
+	It("Verifies Cluster Reset", func() {
+		testcase.TestClusterReset()
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Verifies Ingress", func() {
+		testcase.TestIngress(true, true)
+	})
+
+	It("Verifies Daemonset", func() {
+		testcase.TestDaemonset(true, true)
+	})
+
+	It("Verifies NodePort Service After Reset", func() {
+		testcase.TestServiceNodePort(false, true)
+	})
+
+	It("Verifies dns access", func() {
```

Comment:
do we really need all those tests? 

dns
nodeport
daemonset
ingress
etc ... ?

Reply:
Yes - the aim was to deploy more workloads after the cluster reset is performed.

Nodeport is set to false before the cluster reset so we want to see where the workload is present after cluster reset.